samples/javascript_nodejs/57.teams-conversation-bot/bots/teamsConversationBot.js

```diff
@@ -7,61 +7,114 @@ const {
     TeamsInfo,
     TeamsActivityHandler,
     CardFactory,
-    ActionTypes
-} = require('botbuilder');
-const TextEncoder = require('util').TextEncoder;
+    ActionTypes,
+} = require("botbuilder");
+const TextEncoder = require("util").TextEncoder;
 
 class TeamsConversationBot extends TeamsActivityHandler {
     constructor() {
         super();
+
         this.onMessage(async (context, next) => {
             TurnContext.removeRecipientMention(context.activity);
             const text = context.activity.text.trim().toLocaleLowerCase();
-            if (text.includes('mention')) {
+            if (text.includes("mention")) {
                 await this.mentionActivityAsync(context);
-            } else if (text.includes('update')) {
+            } else if (text.includes("update")) {
                 await this.cardActivityAsync(context, true);
-            } else if (text.includes('delete')) {
+            } else if (text.includes("delete")) {
                 await this.deleteCardActivityAsync(context);
-            } else if (text.includes('message')) {
+            } else if (text.includes("message")) {
                 await this.messageAllMembersAsync(context);
-            } else if (text.includes('who')) {
+            } else if (text.includes("who")) {
                 await this.getSingleMember(context);
             } else {
                 await this.cardActivityAsync(context, false);
             }
         });
 
         this.onMembersAddedActivity(async (context, next) => {
+            //  console.log(context);
```

Comment:
```suggestion
```

Reply:
Marking this as Resolved as wrong file included in this PR